src/components/EntitlementsPanelForm/EntitlementsPanelForm.test.tsx

```diff
@@ -13,9 +13,9 @@ test("can add entitlements", async () => {
     <EntitlementsPanelForm
       addEntitlements={[
         {
-          entitlement: "can_view",
-          entity: "admins",
-          resource: "group",
+          entitlement_type: "can_view",
+          entity_name: "admins",
+          entity_type: "group",
```

Comment:
In the feature branch most of the entities were a single string whereas in the updated API spec they are objects so I've either updated the objects to match the spec or updated them to be an object instead of a string.